internal/controller/controllers/agent_controller.go

```diff
@@ -304,12 +304,12 @@ func (r *AgentReconciler) populateEventsURL(log logrus.FieldLogger, agent *aiv1b
 	return nil
 }
 
-func (r *AgentReconciler) eventsURL(log logrus.FieldLogger, clusterId, agentId string) (string, error) {
+func (r *AgentReconciler) eventsURL(log logrus.FieldLogger, clusterId, infraEnvId, agentId string) (string, error) {
 	eventsURL := fmt.Sprintf("%s%s/v1/clusters/%s/events?host_id=%s", r.ServiceBaseURL, restclient.DefaultBasePath, clusterId, agentId)
 	if r.AuthType != auth.TypeLocal {
 		return eventsURL, nil
 	}
-	eventsURL, err := gencrypto.SignURL(eventsURL, clusterId)
+	eventsURL, err := gencrypto.SignURL(eventsURL, infraEnvId, gencrypto.InfraEnvKey)
```

Comment:
Why did this events url change to infra env if the other is still using cluster?

Reply:
All the operations related to a host need to have infra env.
With v2 and late binding we will probably need to change this URL based on the current status, for example if host is unbound then we will need to have infra env and host id, when it's bound we will search by host id and cluster. 
So there will be work here once we move to `/v2/events` api